pgxscan/pgxscan.go

```diff
@@ -55,12 +55,18 @@ func ScanAll(dst interface{}, rows pgx.Rows) error {
 // See dbscan.ScanOne for details.
 func ScanOne(dst interface{}, rows pgx.Rows) error {
 	err := dbscan.ScanOne(dst, NewRowsAdapter(rows))
+	if NotFound(err) {
+		return errors.WithStack(pgx.ErrNoRows)
+	}
 	return errors.WithStack(err)
 }
 
 // NotFound is a wrapper around the dbscan.NotFound function.
 // See dbscan.NotFound for details.
 func NotFound(err error) bool {
+	if errors.Is(err, pgx.ErrNoRows) {
+		return true
+	}
 	return dbscan.NotFound(err)
```

Comment:
I think we can remove the call to `dbscan.NotFound()` since we always replace dbscan err with `pgx.ErrNoRows` and it should not break the backward compatibility. What do you think? 
Also, it would be good to update the function docs to state that it checks `pgx.ErrNoRows` error.

Reply:
Yup good idea

Reply:
Done